wave_vision/CMakeLists.txt

```diff
@@ -4,40 +4,52 @@ PROJECT(wave_vision)
 # DEPENDENCIES
 FIND_PACKAGE(OpenCV REQUIRED)
 FIND_PACKAGE(Eigen3 REQUIRED)
+FIND_PACKAGE(Boost 1.40 COMPONENTS system filesystem REQUIRED)
```

Comment:
Because C++ suprisingly doesn't have a  `rmdir`. I found sources that strongly recommended for `boost::filesystem::remove_all()` function. But I found another implementation that uses standard C++. Will remove this.

In regards to the FindBoost module, do you think it is a good policy to have our own copies of the modules even though they may exist on the system?